frontend/src/shared/cohort.utils.ts

```diff
@@ -26,19 +26,10 @@ export function getCohortDescription(cohort: CohortConfig) {
 export function getCohortParticipants(
   participants: ParticipantProfile[],
   cohortId: string,
-  countObsoleteParticipants = true, // participants who left the experiment
   includeParticipantsPendingTransferToCohort = true,
 ) {
   let participantList = participants;
 
-  // If not counting obsolete, filter out participants
```

Comment:
What do you think about leaving this in? Even though it won't be applicable (since the field is not accessible via frontend), the field still exists in the backend data structure and someone might need it in the future?

Reply:
Done.